compute/instances/how-to/migrate-routed-ips.mdx

```diff
@@ -13,11 +13,11 @@ categories:
   - compute
 ---
 
-During deployment, each Instance gets a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via an underlying internal network, without public IPs.
+During deployment, each Instance created before April 2024 used to get a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via an underlying internal network, without public IPs.
```

Comment:
```suggestion
Prior to April 2024, all Instances got a NAT IP from their hosting node, which was routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. This was in addition to the public (flexible) IP. The NAT IP let Instances communicate via an underlying internal network, without public IPs. However, if the underlying hypervisor changed, the NAT IP changed also. This could happen when you restarted your Instance. 
```